src/components/pipeline/Pipeline.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import IconButton from '@material-ui/core/IconButton';
+import EditIcon from '@material-ui/icons/Edit';
+import AddIcon from '@material-ui/icons/Add';
+import DeleteIcon from '@material-ui/icons/Delete';
+import Tooltip from '@material-ui/core/Tooltip';
+import LaunchIcon from '@material-ui/icons/Launch';
+
+import Main from '../common/Main';
+import Table from '../common/Table';
+import {
+  PIPELINE_DELETE_BUTTON_CLASS,
+  PIPELINE_EDIT_BUTTON_CLASS,
+  PIPELINE_EXECUTION_BUTTON_CLASS,
+  PIPELINE_NAME_CLASS,
+  PIPELINE_TABLE_ID,
+  PIPELINE_ADD_BUTTON_ID,
+  buildPipelineRowClass,
+} from '../../config/selectors';
+
+const styles = (theme) => ({
+  formControl: {
+    width: theme.spacing(15),
+    margin: theme.spacing(1, 0),
+  },
+  content: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  infoAlert: {
+    margin: theme.spacing(2),
+  },
+  addButton: {
+    color: theme.palette.secondary.main,
+    backgroundColor: theme.palette.primary.main,
+    position: 'fixed',
+    right: theme.spacing(2),
+    bottom: theme.spacing(2),
+    '&:hover, &.Mui-focusVisible': {
+      backgroundColor: theme.palette.primary.main,
+    },
+  },
+});
+
+const mockPipeline = {
```

Comment:
since you use this data in two places, what I would do is within `/app/data`, create a file `mockPipelineData.js`, and put it there... then import it into these files. this will also make these files shorter/easier to manage.